.github/workflows/windows_x64_build.yml

```
@@ -15,12 +15,12 @@ jobs:
     runs-on: windows-latest
     steps:
       - name: checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: setupt msvc console
```

```suggestion
      - name: setup msvc console
```
If we're in here tweaking?